charts/argo-cd/values.yaml

```diff
@@ -161,7 +161,7 @@ configs:
     url: ""
 
     # -- The name of tracking label used by Argo CD for resource pruning
-    # @default -- Defaults to app.kubernetes.io/instance
+    # @default -- Defaults to argocd.argoproj.io/instance
```

Comment:
Can we remove that line completely and run helm-docs in the scripts folder? This way, the default is automatically updated from the real value